### ISSUE TITLE
[UX]  Add screen name in playlist edit screen

### ISSUE THREAD

**Issue by onemustpersist** [usetrmnl/byos_hanami#209]
If 2 screens have the same content you cannot tell the difference between them. Either add the title to the page or the very least add alt/title to the images. The latter also helping with accessibility.

<img width="510" height="764" alt="Image" src="https://github.com/user-attachments/assets/b35e97d9-86db-4dc2-bded-24ff2167c551" />

**Comment by bkuhlmann** [comment]
Thanks, yes, this is a an issue across most pages. It has to do with how partial [htmx](https://htmx.org) responses are being rendered and resolved.

As a temporary workaround, if you refresh the page, you'll see the appropriate title appear. The data is there but need to make sure this happens when rendering HTML fragments too.
